src/ostree/ot-builtin-find-remotes.c

```diff
@@ -232,14 +232,14 @@ ostree_builtin_find_remotes (int            argc,
   /* Build the array of OstreeRepoFinder instances */
   if (opt_finders != NULL)
     {
-      g_auto(GStrv) finders_strings = NULL;
+      g_autofree const char **finders_strings = NULL;
```

Comment:
This will leak though, right? This will only free the array itself, but not each member, like `g_strfreev`.

Reply:
Oops. I guess there's no way to make it `const` then.